src/lib/modules/glimpse-highlight.js

```diff
@@ -4,5 +4,6 @@ var hljs = require('../../../node_modules/highlight.js/lib/highlight.js');
 
 hljs.registerLanguage('cs', require('../../../node_modules/highlight.js/lib/languages/cs'));
 hljs.registerLanguage('sql', require('../../../node_modules/highlight.js/lib/languages/sql'));
+hljs.registerLanguage('javascript', require('../../../node_modules/highlight.js/lib/languages/javascript'));
```

Comment:
Yes.  I've created refactoring issue #95 for that, as this PR is big enough as it is.